js/navigation-search.js

```diff
@@ -112,7 +101,9 @@ bcpl.navigationSearch = (($) => {
 			clearTimeout(resizeTimer);
 			resizeTimer = setTimeout(() => {
 				$menu.addClass('animatable');
-				callback();
+				if (callback) {
```

Comment:
should this be if (callback && typeof callback === 'function') ... i know this was the safest way at one point, but maybe it's overkill

Reply:
You're correct, I was just being lazy. Fixed.